chains/lukso/metadata.yaml

```
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.execution.mainnet.lukso.network/api
+    family: blockscout
+    name: LUKSOExplorer
```

nit based on the name in the explorer home page
```suggestion
    name: LUKSO Mainnet Explorer
```

---

Done